fe/fe-core/src/main/java/com/starrocks/lake/LakeTable.java

```diff
@@ -131,4 +133,22 @@ public Runnable delete(boolean replay) {
     public AlterJobV2Builder alterTable() {
         return new LakeTableAlterJobV2Builder();
     }
+
+    @Override
+    public Map<String, String> getProperties() {
+        Map<String, String> properties = super.getProperties();
+        if (tableProperty != null) {
+            StorageInfo storageInfo = tableProperty.getStorageInfo();
+            if (storageInfo != null) {
+                // enable_storage_cache
+                properties.put(PropertyAnalyzer.PROPERTIES_ENABLE_STORAGE_CACHE,
```

Comment:
Can we have a generic implementation that copies all necessary properties automatically instead of hardcode one by one, so that whenever a new property is added, it will be copied automatically.

Reply:
We can refactor it later.